geode-management/src/main/java/org/apache/geode/management/operation/RebalanceOperation.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.management.operation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import org.apache.geode.annotations.Experimental;
+import org.apache.geode.management.api.ClusterManagementOperation;
+import org.apache.geode.management.runtime.RebalanceResult;
+
+@Experimental
+public class RebalanceOperation implements ClusterManagementOperation<RebalanceResult> {
+  public static final String REBALANCE_ENDPOINT = "/operations/rebalance";
+  private List<String> includeRegions = new ArrayList<>();
+  private List<String> excludeRegions = new ArrayList<>();
+  private boolean simulate;
```

Comment:
Are we supporting "simulate"? I thought I saw you remove some deadcode earlier in this review because "simulate" was always false?

Reply:
Currently the code allows uses to pass "null" in the constructor and the list settors. Is that a legal value for these lists?

Reply:
yes, `simulate` is supported.  just removed duplicate code.  updated setters to accept null (as if empty list was passed).  removed constructor.